README.md

```diff
@@ -26,11 +26,11 @@ Projects listed on this page are open source that Microsoft maintain or contribu
 |  | [AKS Engine](https://github.com/Azure/aks-engine) <br> ([releases](https://github.com/Azure/aks-engine/releases)) | Self-managed clusters on Azure | Azure: winding down <br> API: N/A | [#aks-engine-users](https://kubernetes.slack.com/archives/CU3N85WJK) <br> [GitHub issues](https://github.com/Azure/aks-engine/issues) | <li>[Azure Stack Hub](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-kubernetes-aks-engine-overview)</li> |
```

Comment:
Should we remove aks engine here or at least add a note about the deprecation, like https://github.com/Azure/aks-engine#project-status